apps/web/app/app.dub.co/(dashboard)/settings/tokens/page-client.tsx

```diff
@@ -50,8 +51,12 @@ export default function TokensPageClient() {
               mutate();
               toast.success("Successfully created a new token!");
             } else {
-              const errorMessage = await res.text();
-              toast.error(errorMessage || "Something went wrong");
+              const errorResponse = await res.json();
+              toast.error(
+                errorResponse.error.message ||
+                  errorResponse ||
+                  "Something went wrong",
+              );
```

Comment:
I'm curious about when `errorResponse` is a string and when it's an object.

Reply:
What if we do this?
```typescript else {
              let errorResponse = await res.json();
              if (errorResponse.error.message) {
                toast.error(errorResponse.error.message);
              } else {
                errorResponse = await res.text();
                toast.error(errorResponse || "Something went wrong");
              }
            }

Reply:
@anirbanpaulcom In which case does the API return text vs JSON? Can you clarify?

Reply:
I was getting a text error response for Zod's error.